python/kserve/kserve/model.py

```diff
@@ -169,9 +170,7 @@ def postprocess(self, response: Union[Dict, ModelInferResponse]) -> Dict:
         :param response: Dict|ModelInferResponse passed from predict handler
         :return: Dict
         """
-        if isinstance(response, ModelInferResponse):
```

Comment:
need to check whether the original request is REST or gRPC by examining the headers